source/macos/AdaptiveCards/AdaptiveCards/Views/ACRFactTextField.swift

```diff
@@ -42,9 +44,13 @@ class ACRFactTextField: NSView {
         labelTextField.alignment = .left
     }
     
-    func setupTitle() {
-        // TODO: Make it get all properties from host config
-        labelTextField.font = NSFont.boldSystemFont(ofSize: 12)
+    func setupTitleString() {
+//         TODO: Make it get all properties from host config
```

Comment:
Maybe this method can accept a param `fontSize` with default value 12. By doing this, We can potentially avoid dependency on HostConfig for creating `ACRFactTextField`.
```suggestion
    func setupTitleString(with fontSize: CGFloat = 12) {
//         TODO: Make it get all properties from host config
```
The font size can be calculated with hostConfig in the calling place.